recipes-bsp/partition/qcom-partition-confs_1.0.bb

```diff
@@ -27,6 +27,15 @@ PARTCONF ?= ""
 PARTCONF:qcm6490 ?= "qcm6490-partitions.conf"
 PARTCONF:qcs9100 ?= "qcs9100-partitions.conf"
 
+# For machines with a published cdt file, let's make sure we flash it
+fixup_cdt() {
+    sed -i '/name=cdt/ s/$/ --filename=cdt.bin/' ${S}/${PARTCONF}
+}
+
+do_compile:prepend:qcs6490-rb3gen2-core-kit() {
```

Comment:
@ndechesne Based on the information from https://docs.qualcomm.com/bundle/publicresource/topics/80-70015-254/how_to.html, CDTs for core and vision kits differ (I've compared hashes). Could we also introduce the one for Vision Kit?

This probably will also require a new machine file for `qcs6490-rb3gen2-vision-kit`

Reply:
Yes, they are different. This is why I prepared the recipe so that we can add more MACHINES in here. There will be a -vision .conf file, we just haven't added it yet (see how it is in the 1.x / legacy branch, https://github.com/quic-yocto/meta-qcom-hwe/tree/kirkstone/conf/machine)

Reply:
As we currently use `qcs6490-rb3gen2-core-kit` machine for both kits (I'm testing that build on Vision Kit, for example), my concern is if it can break boot if we deliberately flash wrong CDT?

Reply:
hmm. interesting.. we are shipping only 1 DTB, and I would expect the CDT to basically assist in the DTB selection process, nothing else. So I am pretty sure that you can use either one of the 2 CDT , and it will work. Do you want to try?

Reply:
I've tested both CDTs no my Vision KIt and confirm that both boot

Reply:
Just to note: I still consider that changing CDT for the mezzanines is a mistake. What if the user adds industrial mezz on top of the Vision Kit? Which CDT should be flashed in such a case?

Reply:
We are only using one DTB in the end via the dtb fat image, since we don't have the extra device-id logic in upstream, that is why the generated image is working fine on both.

We will have to create another machine conf for vision kit as well, which will have its own CDT.

Reply:
> Just to note: I still consider that changing CDT for the mezzanines is a mistake. What if the user adds industrial mezz on top of the Vision Kit? Which CDT should be flashed in such a case?

Yes, I also don't agree on using CDT here, as this is just a mezzanine in the end, but I don't think there is another mechanism to identify if the mezzanine is available at runtime atm.

Reply:
My point is that even with CDT we can not identify the mezzanine (and we have no guarantee that there is just a single mezz). The updated boot-selection mechanism should account for that by e.g. allowing a user to manually name attached mezzanines.